Source/ARTURLSessionServerTrust.m

```diff
@@ -20,7 +20,29 @@ @implementation ARTURLSessionServerTrust
 - (instancetype)init:(dispatch_queue_t)queue {
     if (self = [super init]) {
         _queue = queue;
-        _session = [NSURLSession sessionWithConfiguration:[NSURLSessionConfiguration defaultSessionConfiguration] delegate:self delegateQueue:nil];
+        NSURLSessionConfiguration *config = [NSURLSessionConfiguration ephemeralSessionConfiguration];
+#if TARGET_OS_IOS
```

Comment:
```suggestion:-0+19
if(@available(iOS 18.5, macOS 10.12, tvOS 13.0, *)) {
            config.TLSMinimumSupportedProtocolVersion = tls_protocol_version_TLSv12;
            } else {
            config.TLSMinimumSupportedProtocol = kTLSProtocol12;
        }
```

Reply:
OMG, "I'm gonna downvote myself" 😂 This one is definitely better!

Reply:
Fixed in https://github.com/ably/ably-cocoa/pull/1120/commits/b1186870ddbc4b48010cc90576947dbf9f9fc706